lib/parse.cpp

```diff
@@ -655,7 +655,7 @@ bool XML_PARSER::parse_int(const char* start_tag, int& i) {
         }
     }
     errno = 0;
-    int val = strtol(buf, &end, 0);
+    int = strtol(buf, &end, 0);
```

Comment:
Build failed because of this with next error:
```
1>..\lib\parse.cpp(658): error C2513: 'int' : no variable declared before '='
1>..\lib\parse.cpp(666): error C2065: 'val' : undeclared identifier
```

Reply:
Just updated now. Was working on it same time. Thanks.